server/routes.py

```diff
@@ -0,0 +1,179 @@
+import uuid, jwt
+from server import app, db
+from server.jwt.jwt_util import token_required
+from flask import jsonify, request, make_response
+from werkzeug.security import generate_password_hash, check_password_hash
+import datetime
+from server.models import User, Book
+
+@app.route('/user', methods=['GET'])
+@token_required
+def get_all_users(current_user):
+
+    if not current_user.admin:
+        return jsonify({"message": "No permisson for the operation"})
+    users = User.query.all()
+    output = []
+    for user in users:
```

Comment:
In order to avoid this pumping into array and destructuring, you can define dict representation in your model schema, like this:
```
class User(db.Model):
    __tablename__ = 'user'
    id = db.Column(db.Integer, primary_key=True)
    public_id = db.Column(db.String(64), unique=True)
    username = db.Column(db.String(64))
    password = db.Column(db.String(120))
    admin = db.Column(db.Boolean)
    book = db.relationship("Book", backref='author', lazy='dynamic')

    def to_dict(self):
        return {
            'id': self.id,
            'public_id': self.public_id,
            'username': self.username,
            'password': self.password,
            'admin': self.admin
        }
```